examples/root-example/versions.tf

```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = "~> 0.12.0"
```

Comment:
![](https://media2.giphy.com/media/oWA8lD03GUew8/giphy.gif?cid=6104955e31f4f2f52ab3d9863375aef24a95a26f60f9323b&rid=giphy.gif)

gotta love getting on that 0.12 bandwagon